### PR TITLE
games-rpg/adonthell, games-rpg/wastesedge: update to 0.3.8

### PR DESCRIPTION
**Pull request by winterheart**
Update to new versions, remove old.

### CODE REVIEW

File: games-rpg/wastesedge/wastesedge-0.3.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit autotools python-single-r1
+
+DESCRIPTION="role playing game to showcase the adonthell engine"
```
Comment (juippis) [inline games-rpg/wastesedge/wastesedge-0.3.8.ebuild:8]: `Role`

File: games-rpg/wastesedge/wastesedge-0.3.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline games-rpg/wastesedge/wastesedge-0.3.8.ebuild:5]: Please add an empty line here between.

File: games-rpg/wastesedge/wastesedge-0.3.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+DESCRIPTION="role playing game to showcase the adonthell engine"
+HOMEPAGE="http://adonthell.nongnu.org/download/"
+SRC_URI="https://savannah.nongnu.org/download/adonthell/${PN}-src-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="nls"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```
Comment (juippis) [inline games-rpg/wastesedge/wastesedge-0.3.8.ebuild]: Please add an empty line here to make it a bit cleaner.

File: games-rpg/wastesedge/wastesedge-0.3.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+RESTRICT="userpriv"
```
Comment (juippis) [inline games-rpg/wastesedge/wastesedge-0.3.8.ebuild:17]: What's up with this?

File: games-rpg/adonthell/adonthell-0.3.8.ebuild [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools python-single-r1
+
+DESCRIPTION="roleplaying game engine"
```
Comment (juippis) [inline games-rpg/adonthell/adonthell-0.3.8.ebuild:9]: `Roleplaying`

File: games-rpg/adonthell/adonthell-0.3.8.ebuild [hunk 6]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline games-rpg/adonthell/adonthell-0.3.8.ebuild:5]: Empty line between please.

File: games-rpg/adonthell/adonthell-0.3.8.ebuild [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+inherit autotools python-single-r1
```
Comment (juippis) [inline games-rpg/adonthell/adonthell-0.3.8.ebuild:7]: Is autotools being used here?

**Review by juippis — COMMENTED** [review]
No idea why this was "on my table" but looks simple enough for me to process it.

**Comment by winterheart** [comment]
@juippis done

**Review by juippis — APPROVED** [review]
LGTM,
```
 * Messages for package games-rpg/wastesedge-0.3.8:

 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/48x48/apps/wastesedge.png
 *   /usr/share/icons/hicolor/128x128/apps/wastesedge.png
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```

I'll fix it after.

**Pull request closed without merging**